### ISSUE TITLE
Reading order setting doesn't have effect

### ISSUE THREAD

**Issue by brihuega** [tuskyapp/Tusky#3313]
After setting "Reading order" to "oldest first", timeline still shows from newer to oldest.

* * * *
- Tusky Version: 21.0
- Android Version: 12 (One UI Core 4.1)
- Android Device: Samsung Galaxy Tab A7
- Mastodon instance (if applicable): mastodon.social 

- [ x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by nikclayton** [comment]
That's not what it's for: https://mastodon.social/@nikclayton/109839836408666614

**Comment by brihuega** [comment]
OK. It's just a bit complicated to understand.
Scrolling upwards is countet-intuitive for me. Why don't order the messages as requested?

**Comment by n2burns** [comment]
I agree with @brihuega that it would be a nice feature to have the option to flip the order for new toots.

As it stands that setting's title doesn't really describe what it does. Maybe "Load More Position" and "Skip to Bottom"/"Skip to Top"? Maybe someone else can come with something better.

**Comment by Rhialto** [comment]
I have to agree that reading posts "bottom up" is very inconvenient. To lay it on thick, I would even say that the idea that "newest first" is the normal order of reading  is actually the first major problem of Twitter and it should never have been copied to Mastodon.

**Comment by Nairou** [comment]
Came here to find why it wasn't working as well.

I'm fine with the current functionality, but the option needs better wording so it's clear what it actually does.

**Comment by politas** [comment]
Very disappointed that this new setting is only a change to the "Read More" functionality given the text of the option, which seemed quite clear, but isn't. I'm in complete agreement with @Rhialto.

**Comment by Qchristensen** [comment]
Chiming in to agree with everyone else here.  I have been looking patiently for an app which offers the ability to scroll downwards to get to the newest toots and was excited when I saw this option.  I second (seventh?) the request to rename the "reading order" option to perhaps "start from" or something more related to what it actually is, and to please consider an option to change the actual reading order.  In case it's not clear why this would be beneficial:

I like to open the app to start where I left off last time (last read toot - that works great, thank you).  What I'd really like, is to be able to scroll DOWN in order to read newer toots so the first line of a newer toot is directly below the last line of the one I've just read.  Currently, I have to scroll UP.  If I get to a long toot, I have to scroll all the way up to the top of it, then scroll down as I read it, then scroll all the way up past it again to get to the next one - which seems very inefficient.

This is made even harder by the fact that I use larger fonts, so even a not overly long toot can take up more than one screen worth of space - that itself works fine btw, but it definitely exacerbates the inefficient scrolling issue.

(And as a case in point - if someone else has commented on this issue after me - as soon as you finish reading this line, you can just go down a little further, and get to the first line of the next newest comment - that would be perfect in Tusky please!)
